### PR TITLE
Adding opensans locally and fixing font-face mixin functionality

### PR DESCRIPTION
**Pull request by Xymoh**
## Changes 
- Added Open Sans font to be locally imported
- Fixed mixin `@font-face` so that it will work on either online or local imported fonts
- Simplified include declaration of `@font-face` mixin.

## Test
- [ ] Go to ../TDCMCSS/stylesheets/base/_base.scss and add the code at the bottom:
```
p {
  font-family: 'Noto Serif Vithkuqi', sans-serif;
}
```
- [ ] Observe if the font changes for paragraph texts
- [ ] Change the font-family into 'Open Sans' and 'Roboto'
- [ ] Revert the changes and complete the review

### CODE REVIEW

File: stylesheets/abstracts/_mixins.scss [hunk 1]
```diff
@@ -48,27 +48,31 @@
...
-  @each $format in $file-formats {
-    @if ($format !=$file-formats(0)) {
+@mixin font-face($font-family, $font-path) {
+  $file-formats: (woff2: "woff2", woff: "woff", ttf: "truetype", eot: "embedded-opentype", svg: "svg");
+  @if str-slice($font-path, 1, 4) == "http" or str-slice($font-path, 1, 3) == "www" {
+    // If $font-path starts with "http" or "www", it is an external URL
+    @font-face {
+      font-family: $font-family;
+      src: url("#{$font-path}");
+      font-weight: 400t;
```
Comment (mcwiklinski) [inline stylesheets/abstracts/_mixins.scss]: Don't we want to keep using the $font-weight var here?

File: fonts/Open_Sans/.DS_Store
Comment (mcwiklinski) [inline fonts/Open_Sans/.DS_Store:1]: We never need those, but they keep trying to get to our repos 😄

**Review by mcwiklinski — CHANGES_REQUESTED** [review]

File: stylesheets/abstracts/_mixins.scss (hunk 1 above)
Comment (Xymoh) [reply stylesheets/abstracts/_mixins.scss]: You are right, that will give better control over the mixin usage, added $font-weight var again.

File: fonts/Open_Sans/.DS_Store
Comment (Xymoh) [reply fonts/Open_Sans/.DS_Store:1]: It's something related with macos, added a gitignore and deleted the previous one.

Comment (mcwiklinski) [reply fonts/Open_Sans/.DS_Store:1]: Thank you! Bonus points for updating gitignore 👌🏻

**Review by mcwiklinski — APPROVED** [review]

**Pull request merged**